card/lib/generators/card/migration/USAGE

```diff
@@ -6,7 +6,7 @@ Description:
     and choose as migration name import_<filename>.
     
 Example:
-    wagn generate card_migration better_commented_search_cards
+    wagn generate card:migration better_commented_search_cards
```

Comment:
Card (card:) is now a namespace for generator commands as well as a bare command ('card').